app/src/main/java/info/nightscout/androidaps/plugins/general/actions/ActionsFragment.kt

```diff
@@ -233,15 +235,44 @@ class ActionsFragment : DaggerFragment() {
                 actions_canceltempbasal?.visibility = View.GONE
             }
         }
-
+        val activeBgSource = activePlugin.activeBgSource
         actions_historybrowser.visibility = (profile != null).toVisibility()
         actions_fill?.visibility = (pump.pumpDescription.isRefillingCapable && pump.isInitialized && !pump.isSuspended).toVisibility()
         actions_pumpbatterychange?.visibility = pump.pumpDescription.isBatteryReplaceable.toVisibility()
         actions_temptarget?.visibility = (profile != null && config.APS).toVisibility()
         actions_tddstats?.visibility = pump.pumpDescription.supportsTDDs.toVisibility()
 
-        statusLightHandler.updateStatusLights(careportal_canulaage, careportal_insulinage, null, careportal_sensorage, careportal_pbage, null)
+        val shortlabel = skinProvider.activeSkin().description == R.string.lowres_description
```

Comment:
this is dangerous and hard to maintain

Reply:
@MilosKozak LowRes Layout added for Actions plugin and a new function in Skininterface (like overview)
=> is it better now ?

See results with emulator: LowRes enabled according to orientation and Width value (compared to Constants.SMALL_WIDTH)
![AtomEmulator](https://user-images.githubusercontent.com/52934600/97233388-2fadd780-17df-11eb-96f1-73d6a5d25a87.jpg)